apps/docs/src/app/platform/component-docs/platform-forms/radio-group/platform-radio-group-examples/platform-radio-group-list-items-example.component.html

```diff
@@ -1,60 +1,102 @@
         >
         </fdp-radio-group>
-    </fieldset>
-    <p>Example 2: Radio Buttons Types in Compact contentDensity</p>
-    <fieldset fd-fieldset>
+    </fdp-form-field>
+</fdp-form-group>
+<p style="margin-left: 2rem;">Selected value: {{ form1.controls.example1?.value }}</p>
+<br />
+
+<p>Compact Platform Radio Button Created with given List of SelectItem Objects. Using FormGroup and FormControl.</p>
+<fdp-form-group [multiLayout]="true" [formGroup]="form2">
+    <fdp-form-field [id]="'radiolo2'" zone="zLeft" rank="1">
         <fdp-radio-group
-            [name]="'radio2'"
+            [name]="'radiolo2'"
             [contentDensity]="'compact'"
             [list]="items"
             [isInline]="true"
             formControlName="example2"
         >
         </fdp-radio-group>
-    </fieldset>
-</form>
-<div>
-    Example 1 selelected value: {{ customForm.value.example1 }}
-    <br />
-    Example 2 selelected value: {{ customForm.value.example2 }}
-</div>
+    </fdp-form-field>
+</fdp-form-group>
+<p style="margin-left: 2rem;">Selected value: {{ form2.controls.example2?.value }}</p>
+<br />
 
+<p>Platform Radio Button creation and pre-selection based on object data passed.</p>
+<fdp-form-group [multiLayout]="true" [formGroup]="form3" [object]="form3Data">
+    <fdp-form-field #fl3 [id]="'radiolo3'" zone="zLeft" rank="1">
+        <fdp-radio-group [name]="'radiolo3'" [list]="items" [isInline]="true" [formControl]="fl3.formControl">
+        </fdp-radio-group>
+    </fdp-form-field>
+</fdp-form-group>
+<p style="margin-left: 2rem;">Selected value: {{ form3.controls.radiolo3?.value }}</p>
 <br />
-<p>Template Driven Form</p>
-<form class="flex-form">
-    <p>Example 3: Radio Button Group Example with given list of objects, default value selection</p>
-    <fieldset fd-fieldset>
+
+<p>Platform Radio Button created from Invoice objects. Using LookupKey and DisplayKey.</p>
+<fdp-form-group [multiLayout]="true" [formGroup]="form4" [object]="form4Data">
+    <fdp-form-field #fl4 [id]="'radiolo4'" zone="zLeft" rank="1">
         <fdp-radio-group
-            [name]="'radio3'"
-            [value]="'Option 2'"
-            [list]="items"
-            [contentDensity]="'cozy'"
+            [name]="'radiolo4'"
+            [list]="invoiceItems"
             [isInline]="true"
-            [(ngModel)]="favoriteOption"
+            [lookupKey]="'item'"
+            [displayKey]="'itemType'"
+            [formControl]="fl4.formControl"
         >
         </fdp-radio-group>
-    </fieldset>
-    <p>Example 4: Radio Button Group Example with given list of string values</p>
-    <fieldset fd-fieldset>
+    </fdp-form-field>
+</fdp-form-group>
+<p style="margin-left: 2rem;">Preferred Brand: {{ form4.controls.radiolo4?.value }}</p>
+
+<separator></separator>
+<h3>Use With Template Driven Form</h3>
+
+<p>Platform Radio Buttons Created with given List of SelectItem Objects also pre-selection based on Control value</p>
```

Comment:
"Platform radio buttons are created with a given list of SelectItem objects with pre-selection based on control value."

Because this is a description of the component, it's not necessary to capitalize everything. I capitalized "SelectItem" because it seems to be named that way, and would be confusing if it weren't.

Reply:
corrected.